gubernator/main.py

```diff
@@ -34,11 +34,8 @@
 import filters
 
 BUCKET_WHITELIST = {
-    'kubernetes-jenkins',
-    'kube_azure_log',
-    'rktnetes-jenkins',
-    'kubernetes-github-redhat',
-    'kopeio-kubernetes-e2e',
+    re.match(r'gs://([^/]+)', path).group(1)
```

Comment:
Leaving the schema in buckets.json makes it slightly easier to copy-paste into `gsutil ls` and other tools that work with GCS paths.